coordinator-client/build_binaries.sh

```diff
@@ -0,0 +1,4 @@
+docker build -t snark-setup-binaries .
```

Comment:
Update the `coordinator-client/README.md` with instructions to run `sh build_binaries.sh`?

Reply:
Will do!